src/jit/SimdInl.h

```diff
@@ -83,11 +81,14 @@ static void emitExtractLaneSIMD(sljit_compiler* compiler, Instruction* instr)
     if (SLJIT_IS_MEM(args[1].arg)) {
 #if (defined SLJIT_64BIT_ARCHITECTURE && SLJIT_64BIT_ARCHITECTURE)
         sljit_s32 op = (type & SLJIT_32) ? SLJIT_MOV32 : SLJIT_MOV;
+        sljit_emit_op1(compiler, op, args[1].arg, args[1].argw, dstReg, 0);
 #else /* !SLJIT_64BIT_ARCHITECTURE */
-        sljit_s32 op = SLJIT_MOV;
+        if (type == (SLJIT_SIMD_REG_128 | SLJIT_SIMD_ELEM_64)) {
```

Comment:
This is arm32 specific code, not generic 32 bit code. Add an `#elif` case for arm32.